kolibri/utils/cli.py

```diff
@@ -450,13 +225,13 @@ def main(ctx):
 @main.command(cls=KolibriDjangoCommand, help="Start the Kolibri process")
 @click.option(
     "--port",
-    default=OPTIONS["Deployment"]["HTTP_PORT"],
+    default=None,
```

Comment:
Should we update https://kolibri.readthedocs.io/en/latest/access/index.html to comment the possibility of port autodiscovery using ---port=0 ?

Reply:
I think that would be good, yes!